File: fortune/python/fortune.py
import webapp2
import random

class Fortune(webapp2.RequestHandler):
    def get(self):
        self.response.headers['Content-Type'] = 'text/plain'
        N=random.randint(2,11)
        self.response.write('(Python)\nTonight you will get ' + str(N) + ' beers.')

application = webapp2.WSGIApplication([
    ('/', Fortune),
], debug=True)

File: fortune/go/fortune.go
package fosdem2015

import "fmt"
import "math/rand"
import "net/http"

func init() {
	http.HandleFunc("/", fortune)
}

func fortune(w http.ResponseWriter, r *http.Request) {
	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
	
	N := 2 + rand.Intn(10)
	fmt.Fprintf(w, "(Go)\nTonight you will get %d beers.", N)
}
